build.assets/robotest/pr_config.sh

```diff
@@ -67,10 +72,11 @@ EOF
   echo -n $suite
 }
 
-SUITE=$(build_install_suite)
+SUITE=""
+SUITE="$SUITE $(build_install_suite)"
```

Comment:
This change should also occur in nightly.  (for easier commenting of these lines during development.